generator/discovery/update_discovery_doc.sh

```diff
@@ -79,14 +89,27 @@ if [[ -n "${NEW_FILES}" ]]; then
   io::log_red "New resources defined in Discovery Document created new protos:"
   echo "${NEW_FILES}"
   mapfile -t proto_array < <(echo "${NEW_FILES}")
-  io::log_red "Add rest_services definitions to the generator_config.textproto and re-run this script."
+  io::log_red "Adding rest_services definitions to the generator_config.textproto."
   for i in "${proto_array[@]}"; do
     print_service_textproto "${i}"
   done
   git add protos
-  io::log_yellow "Adding new directories to google/cloud/compute/service_dirs.cmake"
+
+  io::log_h2 "Running generate-libraries again with updated generator_config.textproto"
```

Comment:
> again

If `UPDATED_DISCOVERY_DOCUMENT` is set, we could cut off `generate-libraries.sh` after it writes the compute protos.

Then we can run  `build.sh -t generate-libraries-pr || true` here without setting `UPDATED_DISCOVERY_DOCUMENT`?

It seems cleaner to do all the `git add`ing in this script.